cmrtlib/README.md

```diff
@@ -0,0 +1,29 @@
+# Intel(R) C for Media Runtime
+
+
+## Introduction
+
+cmrtlib is a runtime library needed when user wants to execute their own GPU kernels on render engine. It calls iHD media driver to load the kernels and allocate the resources. It provides a set of APIs defined in agnostic/share/cm_rt.h for user to call directly from application.
+
+## License
+
+The Intel(R) Media Driver for VAAPI is distributed under the MIT license.
+You may obtain a copy of the License at:
+
+https://opensource.org/licenses/MIT
+
+## Building and Install
+
+cmrtlib is recommended to be built and installed together with media driver. Please refer to the readme in the root path.
+
+## Backward compatibility
+
+cmrtlib maintains the backward compatibility in both API level and ABI level
+
+## SDK package
+
+You can build your application with the SDK package we maintained in https://01.org/c-for-media-development-package/. In the SDK, we provided the CMake files to build your own application and kernels. You can also find the spec of APIs in the documents of the SDK package.
```

Comment:
This is not what we expect to have! Just above you said that header files are here, media-driver repository. Thus, the primary thing you should explain is: how to use these header files. That's the reason why you need to have cmrt.pc and *.h files installed within media-driver installation. Once done, you need to add this information in this readme. As for the "SDK package" section, please, make sure to formulate it in a way similar to: "**Additionally** you can consider usage of CM SDK package which is described here https://01.org/c-for-media-development-package. It will permit you to do this, this and this (probably compile EU kernels, review additional documentation on how to use C for media, etc.)."

Reply:
I'd prefer to make a short reference:

SDK package can be used with this cmrtlib.  Please refer to SDK package for instructions.

And if SDK package doesn't have instructions to use cmrtlib, file a bug there.